dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/BeamSqlFnExecutor.java

```diff
@@ -280,6 +283,7 @@ static BeamSqlExpression buildExpression(RexNode rexNode) {
         case "SIGN":
           ret = new BeamSqlSignExpression(subExps);
           break;
+        case "SQRT": //SQRT is converted as POWER(i, 0.5)
```

Comment:
if `SQRT` is converted to `POWER`, then we don't even need the `case` statement, right?